docker/Dockerfile

```diff
@@ -43,6 +43,9 @@ RUN mkdir ${HOME}/.jupyter && \
 FROM base AS cpu
 
 # Setup Conda environment
+RUN apt-get update && \
+    apt-get install -y python-pip && \
+    pip install cmake
```

Comment:
why is this needed? isn't it handled within conda?

Reply:
that's a good point, I think it is not necessary since it was added in staging recently https://github.com/microsoft/recommenders/blob/staging/scripts/generate_conda_file.py#L58
FYI @yueguoguo

Reply:
Good catch. It should have been removed.

Reply:
Okay tested it. The conda file takes care of `cmake` and it installs the package, so explicit installation is not needed. Let me remove it.